sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -100,6 +122,109 @@
 		return service.getDeletedUuids(since, user);
 	}
 
+	public Map<Disease, List<FeatureConfigurationIndexDto>> getEnabledFeatureConfigurations(FeatureConfigurationCriteria criteria) {
+		if (criteria == null) {
+			criteria = new FeatureConfigurationCriteria();
+		}
+		criteria.setFeatureTypes(FeatureType.LINE_LISTING);
```

Comment:
Please don't set the feature types here (because this method could also be used for other use cases), but directly where you call this method with the intention of only receiving line listing configurations